### PR TITLE
Add ability to run graphql on a custom domain

### PR DESCRIPTION
**Pull request by blakethepatton**
## Summary

I noticed that when I put graphql on a custom domain using the `route_group_attributes` configuration value that graphiql was no longer able to access the graphql instance. These changes 
enable graphiql to be able to work with a custom domain. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Misc. change (internal, infrastructure, maintenance, etc.)

### Checklist
- [x] Existing tests have been adapted and/or new tests have been added
- [x] Add a CHANGELOG.md entry
- [ ] Update the README.md
- [x] Code style has been fixed via `composer fix-style`

### CODE REVIEW

**Comment by mfn** [comment]
Thanks for the PR!

@blakethepatton you might want to check the tests, practically everything is red (but it looks minor)

@crissi router changes 😬 can you take a look?

**Comment by blakethepatton** [comment]
Oh no! Yeah I'll take a look in the morning.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ CHANGELOG
 
 [Next release](https://github.com/rebing/graphql-laravel/compare/5.1.1...master)
 --------------
+### Changed
```
Comment (mfn) [inline CHANGELOG.md:6]: I would suggest this is `Added` rather, can you adapt it please?

File: src/GraphQLController.php [hunk 2]
```diff
@@ -88,7 +88,7 @@ protected function queryContext(string $query, ?array $params, string $schema)
 
     public function graphiql(Request $request, string $schema = null): View
     {
-        $graphqlPath = '/'.config('graphql.prefix');
+        $graphqlPath = config('graphql.domain').'/'.config('graphql.prefix');
```
Comment (crissi) [inline src/GraphQLController.php:91]: it would not work automatic if it just used the root url,

URL::to('/');

Comment (mfn) [reply src/GraphQLController.php:91]: uh oh, double slash `//` bug or so, right?

**Comment by blakethepatton** [comment]
Hoping to address these requests in the future, closing for now until it's ready for review

**Pull request closed without merging**